docs/import/add_disaster.js

```diff
@@ -325,3 +563,29 @@ function createOptionFrom(innerTextAndValue) {
       .val(innerTextAndValue)
       .prop('id', innerTextAndValue);
 }
+
+/**
+ * Utility function for getting current data. May return undefined if
```

Comment:
when is this called before currentDisaster is set? If it shouldn't be, then just say that?